lib/dynflow/active_job/queue_adapter.rb

```diff
@@ -0,0 +1,34 @@
+module Dynflow
+  module ActiveJob
+    module QueueAdapters
+      # To use Dynflow, set the queue_adapter config to +:dynflow+.
+      #
+      #   Rails.application.config.active_job.queue_adapter = :dynflow
+      class DynflowAdapter
+        extend ActiveSupport::Concern
+
+        class << self
+          def enqueue(job)
+            Foreman.dynflow.world.trigger(JobWrapper, job.serialize)
```

Comment:
should we really say Foreman here? is dynflow not usable outside the foreman scope?